### PR TITLE
Clarify maintenance of auto-instr repos

### PR DESCRIPTION
**Pull request by bhs**
(Per https://github.com/open-telemetry/community/issues/200)

---
This change is [Reviewable](https://reviewable.io/reviews/open-telemetry/oteps/57)

### CODE REVIEW

**Comment by bhs** [comment]
(cc @yurishkuro @bogdandrutu – is this what you were thinking?)

File: text/0001-telemetry-without-manual-instrumentation.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ There are some languages that will have OpenTelemetry support before they have D
 
 ### Governance of the auto-instrumentation libraries
 
-The maintainers for each language `foo` will retain their Approver/Maintainer status and privileges for the `auto-instr-foo` repositories.
+Each `auto-instr-foo` repository must have at least one [Maintainer](https://github.com/open-telemetry/community/blob/master/community-membership.md#maintainer) in common with the main `opentelemetry-foo` language repository. There are no other requirements or constraints about the set of maintainers/approvers for the main language repository and the respective auto-instrumentation repository.
```
Comment (bogdandrutu) [inline text/0001-telemetry-without-manual-instrumentation.md]: Also mentioned clearly that we should support having maintainers of auto that are not maintainers of the language and vice-versa.

Comment (bhs) [reply text/0001-telemetry-without-manual-instrumentation.md]: Sounds good. I made this explicit, @bogdandrutu, and will merge.

**Comment by bogdandrutu** [comment]
What is that "reviewable" button?

**Comment by bhs** [comment]
@carlosalberto @AloisReitbauer @iredelmeier @tigrannajaryan @SergeyKanzhelev @songy23 @reyang @c24t : Can one more of you approve this PR? This repo requires four approvals before merging. (Formally I could override this as a repo admin, but I don't want to bend the rules ;)

Thanks / sorry to pester.

(Of course if there are objections, please object!)

**Comment by bhs** [comment]
@bogdandrutu re

> What is that "reviewable" button?

I have no idea how/why that was added! Not something I did myself. Was it included as a repo or org plugin or something??

**Comment by bhs** [comment]
About "Reviewable": it looks like I used it once or twice in 2015 or something. I have no idea how it turned up in this PR. I just removed it from my personal account in case it slipped in that way:

image


(FYI, it was an alternate code review UI, basically)

**Pull request merged**